inc/featured-media.php

```diff
@@ -305,7 +305,7 @@ function largo_enqueue_featured_media_js( $hook ) {
 			'image_title' => __( 'Featured image', 'largo' ),
 			'gallery_title' => __( 'Featured gallery', 'largo' ),
 			'error_invalid_url' => __( 'Error: please enter a valid URL.', 'largo' ),
-			'error_occurred' => __( 'An error ocurred', 'largo' ),
+			'error_occurred' => __( 'An error occurred', 'largo' ),
```

Comment:
https://github.com/INN/largo/issues/1861 asked that this string have a period after "occurred". Does adding one cause a problem someplace?

Reply:
It shouldn't.